test/utils/Test/Hls/Util.hs

```diff
@@ -126,6 +126,8 @@ ignoreForGhcVersions vers reason
 logFilePath :: String
 logFilePath = "hls-" ++ show ghcVersion ++ ".log"
 
+{- HLINT ignore hlsCommand "Missing NOINLINE pragma" -}
+
 -- | The command to execute the version of hls for the current compiler.
 --
 -- Both @stack test@ and @cabal new-test@ setup the environment so @hls@ is
```

Comment:
Line 137.

Alternatively: do add NOINLINE pragma. Shouldn't really matter, getting the name of the exe should be effectively pure.

Reply:
It might be effectively pure, but it also might not be that cheap, causing it to be evaluated multiple times. This is definitely one that should be fixed.